core-services/prow/02_config/redhat-developer/parodos/_pluginconfig.yaml

```diff
@@ -0,0 +1,20 @@
+approve:
+- commandHelpLink: ""
+  repos:
+  - redhat-developer/parodos
+  require_self_approval: false
+external_plugins:
+  redhat-developer/parodos:
+  - endpoint: http://needs-rebase
+    events:
+    - issue_comment
+    - pull_request
+    name: needs-rebase
+lgtm:
+- repos:
+  - redhat-developer/parodos
+  review_acts_as_lgtm: true
+plugins:
+  redhat-developer/parodos:
+    plugins:
+    - approve
```

Comment:
Please add jira plugin like done [here](https://github.com/openshift/release/blob/dec7a530455aac1f30bdadca5045d685ab8a93a2/core-services/prow/02_config/openshift/_pluginconfig.yaml#L50). It is tracked by https://issues.redhat.com/browse/FLPATH-26

Reply:
Added